usaspending_api/common/query_with_filters.py

```diff
@@ -20,16 +20,16 @@ class _Filter(metaclass=ABCMeta):
     underscore_name = None
 
     @classmethod
-    def generate_query(cls, filter_values: Union[str, list]) -> dict:
+    def generate_query(cls, filter_values: Union[str, list], query_type) -> dict:
```

Comment:
Since the type hints are already present, best to include for the new parameter

Reply:
fair, fixed